esmvalcore/cmor/tables/cmip6/Tables/CMIP6_coordinate.json

```diff
@@ -272,7 +272,7 @@
             "climatology": "", 
             "formula": "", 
             "must_have_bounds": "no", 
-            "out_name": "basin", 
+            "out_name": "sector", 
```

Comment:
just took a closer look at this change - this is interesting, is this an actual error in the CMOR table? We are using the latest version 01.00.33 https://github.com/PCMDI/cmip6-cmor-tables/tree/01.00.33 and that is set as `basin` and it's the same in the Beta version 01.beta.45 - could you please tell me what's the idea behind this change? Also @zklaus - you aware of such a needed change - you keep closer to CMOR much more than me these days :grin:

Reply:
As far as I can see it always has been and remains `basin`. However, I vaguely remember stumbling upon `sector` before---perhaps related with landuse? @dhohn, where is `sector` coming from?

Reply:
Ah, found it. This comes from [CMIP6 Model Output Requirements: File Contents and Format, Data Structure and Metadata
](https://goo.gl/neswPr), specifically note 12 on page 7 (though it is also useful to search for "basin" in the rest of the document).
Basically, the crux of the matter is that the *dimension* is basin, but because strings are stored as character arrays and because dimensional coordinates must be numeric and monotonous, there is *no* dimensional coordinate for the dimension `basin`. Instead, there is an auxiliary coordinate of dimensions `(basin, strlen)`, and that auxiliary coordinate is called `sector` (see [Example  4](https://docs.google.com/document/d/1os9rZ11U0ajY7F8FWtgU4B49KcB59aFlBVGfLC4ahXs/edit#bookmark=id.4d34og8) in the same document).

The CMOR table should remain unchanged at `basin` and if there is a problem in the loading, we should have a close look at the error message and decide the best way forward.

Reply:
excellent find @zklaus - many thanks! What you reckon - shall we wait until we find such an error and think then and there how to fix it?

Reply:
Absolutely. I would leave the table as it should be at `basin` and see where we get. But I fear that @dhohn changed it for a reason, the reason being either a weakness in our treatment of this somewhat unusual coordinate, or a common misinterpretation on the side of the modelers that might necessitate a lot of data fixes. Let's wait and see.

Reply:
Sorry for the confusion this caused. I know nothing about CMOR tables and had no business changing it. This was my naive (and lazy) way of getting all the models to run that I looked at. For some reason they all had the same "mistake" (sector instead of basin). With the above change those models read in fine.
So the proper solution is dataset fixes for all models to rename the variable?

Reply:
I suspect so, but would like to see the actual error and data file before saying definitively.